content/docs/command-reference/add.md

```diff
@@ -6,11 +6,11 @@ Track data files or directories with DVC, by creating a corresponding
 ## Synopsis
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [-f <filename>]
-               targets [targets ...]
+usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
+               [-f <filename>] targets [targets ...]
```

Comment:
this has changed in 1.0 - it's --file now

Reply:
Good catch, will update...

Reply:
Extracted to https://github.com/iterative/dvc.org/issues/1426, please assign a priority - very recent change.